static/app/components/modals/debugFileCustomRepository/appStoreConnect/stepFour.tsx

```diff
@@ -78,13 +78,22 @@ export default StepFour;
 
 const StyledAlert = styled(Alert)`
   div {
-    align-items: center;
+    align-items: flex-start;
+  }
+  @media (min-width: ${p => p.theme.breakpoints[0]}) {
```

Comment:
Should we consider promoting this layout to all Alerts that also contain buttons?

Reply:
yes. thought the same thing.. I will do this change in a follow-up PR 😉